### ISSUE TITLE
Proper cookies syntax using /content API?

### ISSUE THREAD

**Issue by xpose2000** [joelgriffith/browserless#139]
I am trying to figure out the proper syntax for the [content api](https://docs.browserless.io/docs/content.html) to include cookies but have so far run into 400 http_response errors.

This code is in PHP, but I'll also paste the JSON that it creates to make it cross platform.

`$arr 		= array('url' => 'http://www.example.com', 'gotoOptions' => array('waitUntil' => 'networkidle2'), 'cookies' => array(array('name' => 'fghfghfgh', 'value' => 'fghfgh')));`

`{
    "url": "http://www.example.com", 
    "gotoOptions": {
        "waitUntil": "networkidle2"
    }, 
    "cookies": [
        {
            "name": "fghfghfgh", 
            "value": "fghfgh"
        }
    ]
}`

**Comment by joelgriffith** [comment]
Your request looks good... let me check here in a bit!

**Comment by joelgriffith** [comment]
This request below works:

```
curl -X POST \
  http://localhost:3000/content \
  -H 'Content-Type: application/json' \
  -d '{
	"url": "http://www.example.com",
	"gotoOptions": {
		"waitUntil": "networkidle2"
	},
	"cookies": [ { "name": "fghfghfgh", "value": "fghfgh", "domain": "*.example.com" } ]
}'
```

Without a `domain` or `url` property on the cookie(s) you _should_ get an error response with:

`Protocol error (Network.deleteCookies): At least one of the url and domain needs to be specified`

This definitely needs better docs with a full-example, something that I hope to address in the future, but for now going to close this!

**Comment by xpose2000** [comment]
Thanks Joel. I can confirm that the cookies do work as you specified. I wasn't aware domain was required.

However, websites using the [React framework](https://reactjs.org/) that render after initial page load do not seem to have the cookie information included in those AJAX requests.  I don't know why that would matter, but it is definitely the issue. The website in question is ESPN.

I tested the browserless request on a site that uses React and a site that uses jQuery to fire off the ajax requests.  The site with jQuery rendered fine while passing the cookies. The site with React would not render while passing the cookies. 

I can also pass along the examples I used to prove this if you wanted to troubleshoot but they would definitely be need to be kept private.
